students/8/main.js

```diff
@@ -31,13 +31,30 @@ const ex5 = async () => {
 }
 
 const ex6 = async () => {
-    console.log("TODO...")
+    let result = await getPostWithName(11); // 1 is the post id.
+    console.log(result);
 }
 
 //
 // Your functions here...
 //
 
+// Ex 6
+
+const getPostWithName = async (Id) => {
+    let postName = null;
+    try {
+        let post = await axios.get(`https://jsonplaceholder.typicode.com/posts/${Id}`);
+        let userdata = await axios.get(`https://jsonplaceholder.typicode.com/users/${post.data.userId}`);
+        retval2 = { name: userdata.data.name}
```

Comment:
retval2 looks kinda dorky here... I would probably get rid of it and do something like this:

`postName = Object.assign(post.data, { name: userdata.data.name})`

Reply:
yeah it does